pkg/sfu/buffer/buffer.go

```diff
@@ -445,7 +467,13 @@ func (b *Buffer) getExtPacket(rawPacket []byte, rtpPacket *rtp.Packet, arrivalTi
 		}
 		ep.Payload = vp8Packet
 		ep.KeyFrame = vp8Packet.IsKeyFrame
-		ep.TemporalLayer = int32(vp8Packet.TID)
+		if ep.DependencyDescriptor == nil {
+			ep.TemporalLayer = int32(vp8Packet.TID)
+		} else {
+			// vp8 with DependencyDescriptor enabled, use the TID from the descriptor
```

Comment:
oh wow, VP8 can be used with DD too, nice!! I guess DD is pretty much replacement for RTP Frame Marking?

Reply:
@boks1971 yeah, I think DD will be use for all codecs, even with e2e crypto. But not sure it will completely deprecate Frame Marking, need check that